modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase

```diff
@@ -94,13 +94,75 @@ definition {
 	}
 
 	@description = "This is a test for COMMERCE-6019. EditAndRemoveAnOptionWithValues"
-	@ignore = "true"
 	@priority = "3"
 	test EditAndRemoveAnOptionWithValues {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6019 EditAndRemoveAnOptionWithValues
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Options");
+
+		CPCommerceCatalog.newOptionsEntry(
+			optionFieldType = "Select from List",
+			optionName = "Color");
+
+		Button.clickSave();
+
+		AssertElementPresent(locator1 = "Message#SUCCESS");
+		Check.checkToggleSwitch(
+			key_toggleSwitchLabel = "Use in Faceted Navigation",
+			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+		CPCommerceCatalog.addOptionValues(
+			optionEntryName = "Color",
+			optionValuesList = "Black:1");
+
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Products");
+
+		CPCommerceCatalog.newProductsEntry(
+			productCatalog = "Master",
+			productName = "Simple T-Shirt",
+			productType = "Simple");
+
+		CommerceEntry.gotoMenuTab(menuTab = "Options");
+
+		CommerceRelations.assignSingleRelationToProductsEntry(
+			entryName = "Color",
+			title = "Add Option");
+
+		CommerceNavigator.gotoEntry(entryName = "Color");
+
+		CommerceNavigator.gotoEntry(entryName = "Black");
+
+		Type(
+			key_fieldLabel = "Name",
+			locator1 = "TextInput#GENERIC_TEXT_INPUT",
+			value1 = "red");
+
+		Type(
+			key_fieldLabel = "Key",
+			locator1 = "TextInput#GENERIC_TEXT_INPUT",
+			value1 = "red");
+
+		CommerceEntry.saveEntry();
+
+		Navigator.openURL();
+
```

Comment:
Here you have to use Navigator to get back to the home page as you cannot exit out of the Commerce Entry once going to it on line 137